src/components/contactBottomHeaderArea.js

```diff
@@ -0,0 +1,54 @@
+import React from "react"
+import "./contactBottomHeaderArea.scss"
+
+export default function ContactBottomHeaderArea() {
+    const bottomLeft = (
+        <form>
+            <div class="form-row">
+                <input type="text" placeholder="First Name" value="" />
+                <input type="text" placeholder="Last Name" value="" />
+            </div>
+            <div class="form-row"> 
+                <input type="text" placeholder="Email" value="" />
```

Comment:
I know it is probably setup this way in the production, but I'd like you to change this to be an `email` type instead of `text`, that way we get some basic validation out of the box.

```suggestion
                <input type="email" placeholder="Email" value="" />
```